guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The <code>arcgis.learn</code> module includes <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#fullyconnectednetwork" target="_blank">FullyConnectedNetwork</a> and <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.html#mlmodel" target="_blank">MLModel</a> to train machine learning and deep learning models on tabular data. Tabular data can be any vector data in the form of a <a href="https://developers.arcgis.com/python/guide/working-with-feature-layers-and-features/" target="_blank">feature layer</a> or <a href="https://developers.arcgis.com/python/guide/introduction-to-the-spatially-enabled-dataframe/" target="_blank">spatially enabled dataframe</a>. Explanatory variables can take the form of fields in the attribute table of the training features, raster datasets, and distance features used to calculate proximity values for use as additional variables. In addition to validation of model performance based on the training data, predictions can be made to <u>produce</u> either features or a prediction raster.  Predictions can be performed for both categorical variables (classification) and continuous variables (regression). </p><p><u>The </u><code>FullyConnectedNetwork</code> model is based on deep neural networks, whereas <u>the</u> <code>MLModel</code> uses machine learning algorithms to train models. </p><p>The <code>MLModel</code> allows you to use any <a href="https://scikit-learn.org/stable/supervised_learning.html#supervised-learning" target="_blank">regression</a> or <a href="https://scikit-learn.org/stable/supervised_learning.html#supervised-learning" target="_blank">classification</a> model from scikit-learn, whereas <code>FullyConnectedNetwork</code> allows you to train a fully trained deep neural network with a specified number of layers.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>